egs/yesno/ASR/transducer/jointer.py

```diff
@@ -0,0 +1,55 @@
+# Copyright    2021  Xiaomi Corp.        (authors: Fangjun Kuang)
+#
+# See ../../../../LICENSE for clarification regarding multiple authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+class Jointer(nn.Module):
```

Comment:
Is it Jointer or Joiner?

Reply:
I find that both names are used.

## Joiner
- https://research.fb.com/wp-content/uploads/2021/05/Deep-Shallow-Fusion-for-RNN-T-Personalization.pdf
- https://www.cc.gatech.edu/classes/AY2021/cs7643_spring/assets/L24_rnnt_asr_tutorial_gt.pdf

## Jointer
- https://arxiv.org/pdf/2011.01576.pdf

---

I was using Joiner but I always type jointer so jointer is used. 

Do you think Joiner is more suitable here?

Reply:
I thought it’s a typo but now I find it’s actually a valid word 👀 just ignore my comment 😉

https://en.m.wikipedia.org/wiki/Jointer